### PR TITLE
Set glog levels to control verbosity

### PR DESCRIPTION
**Pull request by sknot-rh**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

With this, we can control the verbosity using the `--verbosity` parameter which is already present.

### CODE REVIEW

File: kafka_exporter.go [hunk 1]
```diff
@@ -33,6 +33,13 @@ const (
 	clientID  = "kafka_exporter"
 )
 
+const (
+	OFF = 0
```
Comment (ppatierno) [inline kafka_exporter.go:37]: what this OFF = 0 is? When on `glog` you don't use the verbosity function, it just prints the message provided to `Info` function so I would use INFO = 0 as default and DEBUG = 1 and TRACE = 2.

Comment (sknot-rh) [reply kafka_exporter.go:37]: I thought it would be good to have a possibility to disable info logging.

**Comment by danielqsj** [comment]
LGTM， @sknot-rh please make a rebase :)

**Pull request merged**